servicetalk-dependencies/build.gradle

```diff
@@ -125,3 +132,11 @@ if (!!findProperty("signingKey") && !!findProperty("signingPassword")) {
     sign publishing.publications.mavenJava
   }
 }
+
+def addDependency(Node dependencies, String groupId, String artifactId, String version, String classifier) {
```

Comment:
Perhaps make `version` and `classier` optional and move to plugin utils?

Reply:
This method is required for version with classifier, they are not optional.
We don't need to share it with any other `gradle` file bcz it's needed only for `servicetalk-dependencies`. If we have a use-case later, we can move it then.